cosmic/src/evolv2.f

```diff
@@ -465,11 +464,7 @@ SUBROUTINE evolv2(kstar,mass,tb,ecc,z,tphysf,
 *
 * On the first entry the previous timestep is zero to prevent mass loss.
 *
-      if(using_cmc.eq.1)then
-          dtm = 0.d0
-      else
-          dtm = 1.d0
-      endif
+      dtm = 0.d0
       delet = 0.d0
```

Comment:
@katiebreivik @astroJeff  this is the main thing I need to make sure is OK to do.